mongodb/accounts.go

```diff
@@ -29,17 +29,17 @@ func init() {
 }
 
 type Account struct {
-	Type             AccountType
-	Key              thread.PubKey
-	Secret           thread.Identity
-	Name             string
-	Username         string
-	Email            string
-	Token            thread.Token
-	Members          []Member
-	BucketsTotalSize int64
```

Comment:
This is no longer needed here. That means existing accounts will not be penalized existing stored data, but I think that's the cleaner approach anyway, rather than immediately demanding payment.

Reply:
Fantastic, this was my number one question with all this stuff, and you covered it here. This is a good call.